aspnet/hosting/aspnet-core-module.rst

```diff
@@ -0,0 +1,155 @@
+| stdoutLogFile             | | Optional string attribute.                       |
+|                           | |                                                  |
+|                           | | Specifies the relative or absolute file path for |
+|                           | | which **stdout** and **stderr** from the process |
+|                           | | specified in **processPath** will be logged.     |
+|                           | | Relative paths are relative to the root of the   |
+|                           | | site. Any path starting with '.' will be         |
+|                           | | relative to the site root and all other paths    |
+|                           | | will be treated as absolute paths.               |
+|                           | |                                                  |
+|                           | | The default value is ``httpplatform-stdout``.    |
++---------------------------+----------------------------------------------------+
+| forwardWindowsAuthToken   | | True or False.                                   |
+|                           | |                                                  |
+|                           | | If  true, the token will be forwarded to the     |
+|                           | | child process listening on %HTTP_PLATFORM_PORT%  |
+|                           | | as a header 'X-IIS-WindowsAuthToken' per request.|
+|                           | | It is the responsibility of that process to call |
+|                           | | CloseHandle on this token per request.           |
+|                           | |                                                  |
+|                           | | The default value is false.                      |
++---------------------------+----------------------------------------------------+
+
+Child Elements
+^^^^^^^^^^^^^^^
+
++---------------------------+----------------------------------------------------+
+| Attribute                 |     Description                                    |
++===========================+====================================================+
+| environmentVariables      | | Configures **environmentVariables** collection   |
+|                           | | for the process specified in **processPath**.    |
++---------------------------+----------------------------------------------------+
+
+ASP.NET Core Module app offline
+-------------------------------
+
+If you place a file with the name  *app_offline.htm* at the root of a web application directory, the ASP.NET Core Module will shut-down the application and stop processing any new incoming requests for that application. The ASP.NET Core Module will respond to all requests for dynamic pages in the application by sending back the content of the  *app_offline.htm*  file. For example, you might want to return a "site under construction" or a "down for maintenance" page.
```

Comment:
Nit: Are "any" and "new" unnecessary? ... i.e., an incoming request will be declined whether new or not ... and the last mention of "application" might be superfluous, as well.

> ... the ASP.NET Core Module will shut-down the application and stop processing incoming requests.

Issue #2 ...

> For example, you might want to return a "site under construction" or a "down for maintenance" page.

This doesn't seem like it's an example of the behavior just described. It's more of a tip, right? Should it say something like ...

> To avoid returning the default content of *app_offline.htm* , you might want to return a "site under construction" or a "down for maintenance" page.

It might also be a good idea to say how to do that. Tell the dev to modify the content of the file itself.

Also note that this language is dangerously similar to it's source: http://weblogs.asp.net/scottgu/426755 ... Personally, I'd rephrase a bit more to avoid needing a citation (but I also realize that Scott's blog is probably owned my MS, in which case it would be a courtesy to cite him).